da/celestia/celestia.go

```diff
@@ -189,30 +196,29 @@ func (c *DataAvailabilityLayerClient) SubmitBatch(batch *types.Batch) da.ResultS
 			},
 		}
 	}
+	blobs = [][]byte{data}
 
-	blockBlob, err := blob.NewBlobV0(c.config.NamespaceID.Bytes(), data)
-	if err != nil {
+	if len(data) > celtypes.DefaultMaxBytes {
```

Comment:
this value should be validated for dymint config as well (we have max bytes for batch there)